php/Blocks/Controls/Multiselect.php

```diff
@@ -53,13 +53,7 @@ public function register_settings() {
 				'label'    => __( 'Choices', 'genesis-custom-blocks' ),
 				'type'     => 'textarea_array',
 				'default'  => '',
-				'help'     => sprintf(
-					'%s %s<br />%s<br />%s',
-					__( 'Enter each choice on a new line.', 'genesis-custom-blocks' ),
-					__( 'To specify the value and label separately, use this format:', 'genesis-custom-blocks' ),
-					_x( 'foo : Foo', 'Format for the menu values. option_value : Option Name', 'genesis-custom-blocks' ),
-					_x( 'bar : Bar', 'Format for the menu values. option_value : Option Name', 'genesis-custom-blocks' )
-				),
+				'help'     => '',
```

Comment:
This is now in the [React file](https://github.com/studiopress/genesis-custom-blocks/pull/62/files#r570563346). 

This is `''` here in PHP so the React file uses it.